modules/mobile/app/helpers/mobile/appointments_cache_interface.rb

```diff
@@ -11,7 +11,7 @@ def fetch_appointments(user:, start_date: nil, end_date: nil, fetch_cache: true)
       search_start_date = [start_date, latest_allowable_cache_start_date].compact.min
       search_end_date = [end_date, earliest_allowable_cache_end_date].compact.max
 
-      if use_cache?(search_start_date, search_end_date, fetch_cache)
+      if fetch_cache?(search_start_date, search_end_date, fetch_cache)
```

Comment:
Renaming this because `use_cache` is a bit ambiguous, given that we still set the cache if we're not fetching from it.